src/dispatch/database/revisions/tenant/versions/2024-04-22_50e99c66e72f.py

```diff
@@ -0,0 +1,64 @@
+"""empty message
+
+Revision ID: 50e99c66e72f
+Revises: 3a33bc153e7e
+Create Date: 2024-04-22 14:19:18.388675
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = '50e99c66e72f'
+down_revision = '3a33bc153e7e'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('assoc_signal_instance_tags',
```

Comment:
It doesn't look like this table was added in code? I would double check the migration was run against an up-to-date database, as I wouldn't expect these migrations to be run given the code changes.

Reply:
These entries still occur with an up-to-date database. I've now manually removed them from this PR.